integrations/slack/src/utils.ts

```diff
@@ -100,3 +102,20 @@ export function getActionNameAndType(actionId: string) {
 
     return { actionName, actionPostType };
 }
+
+export function isSaveThreadMessage(message: string) {
+    const tokens = message.split(' ');
+
+    if (tokens.length > 3) {
+        return false;
+    }
+
+    const [first, second] = tokens;
+
+    // `save` or `save this` or `please save`
```

Comment:
I like this "fuzziness" . for the future we could also just match on /.+save.+/ in the first few words. This doesn't really account for "Our API is now at v2 @GitBook save" which could also happen.